validation/neuvector/pipeline/allinone-pass/Jenkinsfile.nv_allinone_auto_03

```diff
@@ -0,0 +1,10 @@
+#!/groovy
+node('neuvector') {
+    stage('Execution') {
+   withCredentials([string(credentialsId: 'NEUVECTOR_PASSWORD', variable: 'NEUVECTOR_PASSWORD')]) {
+    sh '''
+      sshpass -p "$NEUVECTOR_PASSWORD" ssh neuvector@10.1.5.55 "/mnt/nfs/share/QA/automation/allinone_script.sh $NV_RELEASE_TAG $_DIR_ $JOB_NAME $BUILD_NUMBER"
```

Comment:
I know these run behind the VPN but, I think it would be a good idea for best-practice to put these IPs in a jenkins secret instead